### PR TITLE
Add clarifying sentence

### PR DESCRIPTION
**Pull request by Ana-AlonsoCanizares**

### CODE REVIEW

**Comment by American-Dipper** [comment]
@shsagir - can you review the proposed changes?  

 
IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.  

 

#label:"aq-pr-triaged" 

@MicrosoftDocs/public-repo-pr-review-team

**Comment by shsagir** [comment]
@Ana-AlonsoCanizares Thanks for contacting us with feedback about our product documentation. Depending on the complexity, it could take from a few business days to a few weeks to look further into your question, issue, or suggestion. We'll contact you if we need further information, and you'll receive an email message each time we "@mention" you in a comment. 

Thanks again for submitting your feedback. Your suggestions help improve our documentation.

File: data-explorer/kusto/management/create-tables-command.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Requires [Database user permission](./access-control/role-based-access-control.m
 
 `.create` `tables` *TableName1* ([columnName:columnType], ...) [`,` *TableName2* ([columnName:columnType], ...) ... ] [`with` `(` [`docstring` `=` *Documentation*] [`,` `folder` `=` *FolderName*] `)`]
 
-If any table already exists, the command will return success.
+If any or all specifed tables already exist, the command will return success. Those specified tables which don't exist are created.
```
Comment (shsagir) [inline data-explorer/kusto/management/create-tables-command.md:20]:
```suggestion
If any or all specified tables already exist, the command returns success. Specified tables that don't exist are created.
```

**Review by shsagir — APPROVED** [review]

**Comment by shsagir** [comment]
@Ana-AlonsoCanizares Thank you for your contribution. We've submitted a documentation change based on your suggestion, and the updated article should be live by tomorrow.

#sign-off

File: data-explorer/kusto/management/create-tables-command.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Requires [Database user permission](./access-control/role-based-access-control.m
 
 `.create` `tables` *TableName1* ([columnName:columnType], ...) [`,` *TableName2* ([columnName:columnType], ...) ... ] [`with` `(` [`docstring` `=` *Documentation*] [`,` `folder` `=` *FolderName*] `)`]
 
-If any table already exists, the command will return success.
+If any or all specified tables already exist, the command returns success. Specified tables that don't exist are created.
```
Comment (shsagir) [inline data-explorer/kusto/management/create-tables-command.md:20]:
```suggestion
> [!NOTE]
> If one or more tables with the same (case-sensitive) names as the specified tables already exist in the context of the database, the command returns success without changing the existing tables, even in the following scenarios:
>
> - The specified schema doesn't match the schema of an existing table
> - The `folder` or `docstring` parameters are specified with values different from the ones set in the existing tables
>
> Any specified tables that don't exist are created.
```

**Review by shsagir — APPROVED** [review]

**Comment by shsagir** [comment]
#sign-off

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**